### PR TITLE
[EMB-16] Route dashboard to ember-osf-web

### PR DESCRIPTION
**Pull request by hmoco**
## Purpose
Use of ember to recreate the dashboard, rerouting to ember-osf-web when a user is logged in. Also allow server side filtering and ordering of a user's nodes.

## Changes
Serve ember-osf-web when a user goes to route `/` while logged in.
Remove hard-coded ordering of a user's nodes to allow for custom ordering.
(#TODO: remove dashboard code from osf)

## QA Notes
Test functionality of logged in dashboard, should be similar to current dashboard

## Side Effects
`/` will lead to the Ember-app

## Ticket

https://openscience.atlassian.net/browse/EMB-16

### CODE REVIEW

File: website/views.py [hunk 1]
```diff
@@ -24,7 +23,7 @@
 from website.institutions.views import serialize_institution
 
 from osf.models import BaseFileNode, Guid, Institution, PreprintService, AbstractNode, Node
-from website.settings import EXTERNAL_EMBER_APPS, PROXY_EMBER_APPS, INSTITUTION_DISPLAY_NODE_THRESHOLD, DOMAIN
```
Comment (aaxelb) [inline website/views.py]: Should this also be deleted from settings?

Comment (hmoco) [reply website/views.py]: yeah I think so!

File: api/users/views.py [hunk 2]
```diff
@@ -543,7 +543,6 @@ def get_queryset(self):
         return (
             AbstractNode.objects.filter(id__in=set(self.get_queryset_from_request().values_list('id', flat=True)))
             .select_related('node_license')
-            .order_by('-modified', )
```
Comment (jamescdavis) [inline api/users/views.py]: Please remove the commit for this (and the test) from this PR.

File: website/views.py [hunk 3]
```diff
@@ -142,27 +141,8 @@ def index():
...
-            {'id': inst._id, 'name': inst.name, 'logo_path': inst.logo_path_rounded_corners}
-            for inst in all_institutions
-        ]
-
-        return {
-            'home': True,
-            'dashboard_institutions': dashboard_institutions,
-        }
+    if user_id:
+        return send_from_directory(ember_osf_web_dir, 'index.html')
```
Comment (jamescdavis) [inline website/views.py]: Please leave all the legacy dashboard stuff in place and use waffle to choose between legacy dashboard and Ember dashboard.

Comment (hmoco) [reply website/views.py]: I believe I need a little more than this. As in, do you want a switch that is indefinitely set as either true or false, or a sample, or a flag?

**Comment by hmoco** [comment]
@jamescdavis, was easier to separate the small functionality left in this PR + waffle into another PR. This is now superseded by: https://github.com/CenterForOpenScience/osf.io/pull/8165

**Pull request closed without merging**